lib/gui/pages/main/templates/main.tpl.html

```diff
@@ -3,7 +3,7 @@
     <div class="box text-center relative" os-dropzone="image.selectImageByPath($file)">
 
       <div class="center-block">
-        <svg-icon path="main.selection.getImageLogo() || '../assets/image.svg'"></svg-icon>
+        <svg-icon contents="main.selection.getImageLogo()" path="'../assets/image.svg'"></svg-icon>
```

Comment:
Hmmm, as well as `paths` and `contents` I wonder if it'd be worth also allowing `content` and `path` for the (common) situation where you only want to pass in a single path, not multiple paths? \*shrug\*
(obviously you'd need to explicitly document which order each of these properties are tried in)